Source/WebCore/platform/graphics/DisplayRefreshMonitor.cpp

```diff
@@ -223,7 +223,7 @@ void DisplayRefreshMonitor::displayDidRefresh(const DisplayUpdate& displayUpdate
         setIsPreviousFrameDone(true);
     }
 
-    DisplayRefreshMonitorManager::sharedManager().displayDidRefresh(*this);
+    DisplayRefreshMonitorManager::sharedManager().displayMonitorDisplayDidRefresh(*this);
```

Comment:
displaymonitordisplay?

Reply:
dispayMonitor-displayDidRefresh. Considered `monitorDisplayDidRefresh` but there's possible confusion between the computer-display noun "monitor" and the verb "to monitor".